task_list.sass

```diff
@@ -15,8 +15,8 @@
   cursor: default
   pointer-events: all
 
-.ql-editor ul.task-list > li.checked
+.ql-editor ul.task-list > li.checked-true
```

Comment:
With the way the Attributor is set now, that's what parchment creates.  I think it would be possible to have it set the class to only "checked" but then the task's Delta would be represented weirdly (it wouldn't say whether or not the task's checked).  At least, that's how I interpreted parchment's kinda confusing docs.